assets/js/pages/project/details/members.vue

```diff
@@ -34,10 +73,13 @@
             </v-row>
             <v-row class="align-baseline flex-wrap">
                 <v-col class="flex-md-grow-1" cols="12" md="auto">
-                    <user-search :current-users="currentUsers" :selected.sync="newUsers" :disabled="updating"></user-search>
+                    <user-search :current-users="currentUsers" :selected.sync="newUsers"
+                                 :disabled="updating"></user-search>
                 </v-col>
                 <v-col class="d-flex justify-end flex-md-grow-0" cols="12" md="auto">
-                    <v-btn color="primary" @click="addUsers" :disabled="!newUsers.length || updating">Add selected users</v-btn>
+                    <v-btn color="primary" @click="addUsers" :disabled="!newUsers.length || updating">Add selected
```

Comment:
Is it proper formatting? Maybe put down whole text in new line? Unless there is something I'm missing in community standards :)

Reply:
Autoformatting by phpstorm probably just broke the line at the limit. I'll fix this and move the member menu to separate component. Thanks for catching that!